src/main/java/de/fraunhofer/aisec/cpg/passes/EvaluationOrderGraphPass.java

```diff
@@ -372,6 +380,7 @@ private void createEOG(@Nullable Statement statement) {
 
       // push statement itself
       pushToEOG(statement);
+
```

Comment:
Unnecessary new line :)

Reply:
Tried to use newlines to segment code blocks a bit, as they were all squshed up ;)